optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -33,6 +33,11 @@
     VALID_SET_TYPE = Union[List[lgb.Dataset], Tuple[lgb.Dataset, ...], lgb.Dataset]
 
 
+ELAPSED_SECS_KEY = "lightgbm_tuner:elapsed_secs"
+AVERAGE_ITERATION_TIME_KEY = "lightgbm_tuner:average_iteration_time"
+STEP_NAME_KEY = "lightgbm_tuner:step_name"
+LGBM_PARAMS_KEY = "lightgbm_tuner:lgbm_params"
```

Comment:
Do you think the following is too verbose? I personally find it easier to comprehend as the intended use is clear, but it's not a strong opinion.
```suggestion
LGBM_PARAMS_SYSTEM_ATTR_KEY = "lightgbm_tuner:lgbm_params"
```

Reply:
IMO, the following line is a bit complicated because `system_attr` appears two times (if we ignore the case):

```python
 trial.set_system_attr(_LGBM_PARAMS_SYSTEM_ATTR_KEY, json.dumps(self.lgbm_params))
```

So, I personally prefer `LGBM_PARAMS_KEY`. Instead I add a comment when I define variables:

```python
# Define key names of `Trial.system_attrs`.
_ELAPSED_SECS_KEY = "lightgbm_tuner:elapsed_secs"
...
```

What do you think of these changes?

Reply:
Thanks, the comments indeed help. LGTM!